Misc/NEWS.d/next/Core_and_Builtins/2024-10-07-19-26-50.gh-issue-125038.ffSLCz.rst

```diff
@@ -0,0 +1,2 @@
+Fix SIGSEGV in genexpr after direct changes on gi_frame.f_locals. Patch by
```

Comment:
```suggestion
Fix crash when iterating over a generator expression after direct changes on ``gi_frame.f_locals``. Patch by
```
The exact crash mechanism is OS-dependent and not guaranteed, so let's just say "crash".